corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -224,8 +225,14 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
 
         templateContext: function () {
             var appId = Util.currentUrlToObject().appId;
+            var data = this.options.model.get('data').map(function (datum) {
+                return {
+                    raw: datum,
+                    html: DOMPurify.sanitize(md.renderInline(datum)),
+                };
```

Comment:
Does returning both preclude _case_detail_'s template from ever using the last `else` clause, where it just renders the data? Is there no way to know what the template will actually require from this template? I figured the template and the context were tightly-coupled, and was hopeful we'd be able to specify whether the template needed the raw data or the html. It seems weird it would ever need both

Reply:
Yeah I'm not sure if they are tightly coupled or not, but there's nothing that enforces that coupling.
It might be reasonable to remove the `else` clause, I just separated it to its own thing to be safe. But we'd still need to provide both because it seems like we don't always want to md render and sanitize when there is a [style](https://github.com/dimagi/commcare-hq/pull/29557/files#diff-9ba2478f84098bf46ce7d7af2e93881d529b6e0cc6e425d7857a17f0ac992438R77-R79)--I think anyway, it seems like phone numbers and images should get the raw, and in the future there might be things that want the raw as well. 
I thought of including the corresponding style into the data list as well instead of indexing into the styles list as it does now, but it was too many changes that I didn't know how to test myself.

Reply:
I suspect removing the else clause is okay. The full set of functionality should be testable as long as the QA plan hits each of the detail formats that can be chosen in app manager's case list/detail config.

Reply:
Sounds good, I'll remove it for this one and the other one.